ChaRo-iOS/ChaRo-iOS/Source/Views/Cells/TVC/PostDetailScene/PostAttentionTVC.swift

```diff
@@ -10,132 +10,67 @@ import SnapKit
 
-    private var buttonList : [UIButton] = []
-    
-    private var highwayButton: UIButton = {
-        let button = UIButton()
-        button.setBackgroundImage(UIImage(named: "postAttentionUnselected"), for: .normal)
-        button.setTitle("고속도로", for: .normal)
-        button.titleLabel?.font = UIFont.notoSansMediumFont(ofSize: 14)
-        button.setTitleColor(.gray40, for: .normal)
-        
-        return button
-    }()
-   
-    private var mountainButton: UIButton = {
-        let button = UIButton()
-        button.setBackgroundImage(UIImage(named: "postAttentionUnselected"), for: .normal)
-        button.setTitle("산길포함", for: .normal)
-        button.titleLabel?.font = UIFont.notoSansMediumFont(ofSize: 14)
-        button.setTitleColor(.gray40, for: .normal)
-        button.imageView?.contentMode = .scaleAspectFill
-        return button
-    }()
-    
-    private var beginnerButton: UIButton = {
-        let button = UIButton()
-        button.setBackgroundImage(UIImage(named: "postAttentionUnselected"), for: .normal)
-        button.setTitle("초보힘듦", for: .normal)
-        button.titleLabel?.font = UIFont.notoSansMediumFont(ofSize: 14)
-        button.setTitleColor(.gray40, for: .normal)
-        button.imageView?.contentMode = .scaleAspectFill
-        return button
-    }()
-    
-    private var peopleButton: UIButton = {
-        let button = UIButton()
-        button.setBackgroundImage(UIImage(named: "postAttentionUnselected"), for: .normal)
-        button.setTitle("사람많음", for: .normal)
-        button.titleLabel?.font = UIFont.notoSansMediumFont(ofSize: 14)
-        button.setTitleColor(.gray40, for: .normal)
-        return button
-    }()
+    private let attentionStackView = UIStackView().then {
+        $0.axis = .vertical
+        $0.spacing = 8
+        $0.distribution = .fillEqually
+    }
     
+    private let highwayButton = PostDetailContentButton()
+    private let mountainButton = PostDetailContentButton()
+    private let beginnerButton = PostDetailContentButton()
+    private let peopleButton = PostDetailContentButton()
     
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        setButtonList()
-        configureLayout()
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupConstraints()
+        configureUI()
     }
-
-    override func setSelected(_ selected: Bool, animated: Bool) {
-        super.setSelected(selected, animated: animated)
-
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
     }
     
     public func setAttentionList(list: [Bool]){
-        for i in 0..<list.count{
-            if list[i] {
-                changeToActiveButton(button: buttonList[i])
-                buttonList[i].isUserInteractionEnabled = false
-            }
-        }
+        if list.count != 4 { return }
```

Comment:
```suggestion
        guard list.count >= 4 else { return }
```
guard 문이 더 명시적일 것 같아요. 

이건 옵셔널인데,, 조건이 조금 불안정하긴한데 list를 count를 체크하는 것 말고 인덱스를 검증하는 방법은 어떨까요?

Reply:
guard로 변경했어요~! https://github.com/TeamChaRo/ChaRo-iOS/pull/197/commits/a32331625a35ec0cc29858c5cd7a3d6f37a9e071

@inddoni 인덱스를 검증한다는게 잘 와닿지 않아서 혹시 생각한게 있으면 얘기해 줄 수 있을까요??😊